portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/searchtuning/SearchExperiences.testcase

```diff
@@ -0,0 +1,601 @@
+			JSONUser.addUser(
+				userEmailAddress = "userea${userNumber}@liferay.com",
+				userFirstName = "userfn${userNumber}",
+				userLastName = "userln${userNumber}",
+				userScreenName = "usersn${userNumber}");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea${userNumber}@liferay.com");
+		}
+
+		JSONRole.assignRoleToUser(
+			roleTitle = "Administrator",
+			userEmailAddress = "userea1@liferay.com");
+
+		JSONRole.assignRoleToUser(
+			roleTitle = "User",
+			userEmailAddress = "userea2@liferay.com");
+
+		User.logoutPG();
+
+		User.loginUserPG(
+			password = "test",
+			userEmailAddress = "userea1@liferay.com");
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(name = "Los Angeles Boost");
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.addWidgets(searchPortletList = "Blueprints Options");
+
+		SearchExperiences.selectBlueprintFromOptionsWidget(name = "Los Angeles Boost");
+
+		SearchPortlets.searchEmbedded(searchTerm = "cafe");
+
+		SearchResultPortlet.viewSearchResults(
+			searchAssetTitle = "Cafe",
+			searchAssetType = "Web Content Article");
+
+		User.logoutPG();
+
+		User.loginUserPG(
+			password = "test",
+			userEmailAddress = "userea2@liferay.com");
+
+		AssertElementNotPresent(locator1 = "ControlMenu#APPLICATIONS_MENU_ICON");
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		Portlet.viewNoPermissionPG();
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "cafe");
+
+		SearchResultPortlet.viewSearchResults(
+			searchAssetTitle = "Cafe",
+			searchAssetType = "Web Content Article");
+	}
+
+	test AssertSearchResultsScore {
+		property test.name.skip.portal.instance = "SearchExperiences#AssertSearchResultsScore";
+
+		JSONWebcontent.addWebContent(
+			content = "WC Content",
+			groupName = "Guest",
+			title = "Apple");
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(name = "Score Explanation Blueprint");
+
+		LexiconEntry.gotoEntryMenuItem(
+			menuItem = "Edit",
+			rowEntry = "Score Explanation Blueprint");
+
+		Click(locator1 = "Button#PREVIEW");
+
+		SearchExperiences.searchBlueprintsPreview(searchTerm = "Apple");
+
+		Click(
+			key_searchAssetTitle = "Apple",
+			locator1 = "SearchExperiences#BLUEPRINTS_PREVIEW_SCORE");
+
+		AssertTextEquals(
+			locator1 = "Modal#HEADER",
+			value1 = "Score Explanation");
+
+		AssertVisible(
+			key_searchExplanation = "PerFieldSimilarity",
+			locator1 = "SearchExperiences#BLUEPRINTS_PREVIEW_SCORE_EXPLANATION");
+	}
+
+	test BoostWithElasticsearchQuery {
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(
+			clauseOption = "Custom Clauses",
```

Comment:
why do we use custom clauses here? it looks like the original use case just uses the default https://issues.liferay.com/browse/LPS-125324

Reply:
I think I accidentally wrote custom clauses in the blueprints exploratory doc and copied it over without double checking

Reply:
Although I don't think we have any functional tests that use custom clauses - is this something we should cover?

Reply:
not sure if custom clauses is covered in backend tests already. @xbrianlee would you happen to know?

Reply:
My understanding is that our backend tests don't test with default clauses so by default they're custom clauses. On the frontend, we should assert that we can create a custom clause via the UI though.

Reply:
ah ic, I guess we can create a separate test that asserts that

Reply:
created https://issues.liferay.com/browse/LPS-134327 for this. Not sure which epic to assign to it tho.